JsonDeserializace.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Lekcia12_cvicenie
+{
+    internal class JsonDeserializace
+    {
+        public class WeatherData
+        {
+            [JsonPropertyName("location")]
+            public Location Location { get; set; }
+
+            [JsonPropertyName("current")]
+            public CurrentWeather Current { get; set; }
+
+            [JsonPropertyName("forecast")]
+            public Forecast Forecast { get; set; }
+        }
+
+        public class Location
+        {
+            [JsonPropertyName("name")]
```

Comment:
Je spravne, ze jsi dal vsechny dto objekty do jednoho souboru. V pripade, kdy jsou takhle male tak se to casto dela. Akorat nazev souboru, co je drzi bych dal treba `ForecastApiDto`, nebo neco podobneho.

A take je asi vhodne ty tridy mit samy o sobe a ne v ramci tridy JsonDeserializace.

```suggestion
{
    public class WeatherData
    {
        [JsonPropertyName("location")]
        public Location Location { get; set; }
        
         [JsonPropertyName("current")]
        public CurrentWeather Current { get; set; }

         [JsonPropertyName("forecast")]
        public Forecast Forecast { get; set; }
    }

        public class Location
        {
            [JsonPropertyName("name")]
```